services/QuillLMS/config/initializers/auth/classlink.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Auth
+  module Classlink
+    CLIENT_ID = 'c17370377633256aee86a52043b96d2913e27b36df24'
```

Comment:
No strong feelings on this, but should we make this an ENV variable, too?  If we had a staging Classlink environment, would we have the same `CLIENT_ID` or would it change?